lib/jsdom/living/helpers/strings.js

```diff
@@ -1,5 +1,15 @@
 "use strict";
 
+// https://infra.spec.whatwg.org/#ascii-lowercase
+exports.asciiLowercase = s => {
+  return s.replace(/[A-Z]/g, l => l.toLowerCase());
+};
+
+// https://infra.spec.whatwg.org/#ascii-uppercase
+exports.asciiUppercase = s => {
```

Comment:
Unused?

Reply:
Indeed, but it makes sense to have an upper case counterpart to lower case, no?